lib/main.coffee

```diff
@@ -1,11 +1,30 @@
 {CompositeDisposable} = require('atom')
 
+# The default grammar scopes checked by this linter.
+defaultGrammarScopes = [
+  'text.html.basic'
+]
+
 module.exports =
   config:
     executablePath:
       default: 'tidy'
       title: 'Full path to the `tidy` executable'
       type: 'string'
+    customGrammarScopes:
+      default: []
+      title: 'Custom Grammar Scopes'
+      description: 'A list of extra grammar scopes to lint with Tidy.<br/><br/>
```

Comment:
You should have something like [this](https://github.com/AtomLinter/linter-htmlhint/blob/00417d8335267391920afe16b51e28431a410034/package.json#L16) in there telling users how to find the value to stick in the list.